client/dashboard/leaderboards/top-customers.js

```diff
@@ -57,40 +57,26 @@ export class TopCustomers extends Component {
         const { query } = this.props;
         const persistedQuery = getPersistedQuery( query );
         return map( data, row => {
-            //const { amount, coupon_id, extended_info, orders_count } = row;
-            const { id, total_spend, name, orders_count, avg_order_value, user_id } = row;
-            console.log(row);
-            //const { code } = extended_info;
+            const { id, total_spend, name, orders_count } = row;
 
             const customerUrl = getNewPath( persistedQuery, 'analytics/customers', {
-                // filter: 'single_',
-                // coupons: coupon_id,
-                search: 'name',
+                filter: 'single_customer',
+                customers: id,
             } );
             const customerLink = (
                 <Link href={ customerUrl } type="wc-admin">
-                { name }
+                    { name }
                 </Link>
-        );
+            );
 
-            const ordersUrl = getNewPath( persistedQuery, 'analytics/orders', {
-                filter: 'advanced',
-                customer_id: user_id,
-               // coupon_includes: coupon_id,
-            } );
-            const ordersLink = (
-                <Link href={ ordersUrl } type="wc-admin">
-                { numberFormat( orders_count ) }
-                </Link>
-        );
             return [
                 {
                     display: customerLink,
                     value: name,
                 },
                 {
-                    display: ordersLink,
-                    value: orders_count,
+                    display: orders_count,
```

Comment:
Formatting should go on the display value here:

```js
                {
                    display: numberFormat( orders_count ),
                    value: orders_count,
                },

Reply:
Done